docs/get_started/examples/workflowinterface_pytorch_mnist.rst

```diff
@@ -0,0 +1,353 @@
+                batch_idx * len(data), len(self.train_loader.dataset),
+                100. * batch_idx / len(self.train_loader), loss.item()))
+                self.loss = loss.item()
+                torch.save(self.model.state_dict(), 'model.pth')
+                torch.save(self.optimizer.state_dict(), 'optimizer.pth')
+            self.training_completed = True
+            self.next(self.local_model_validation)
+
+        @collaborator
+        def local_model_validation(self):
+            self.local_validation_score = inference(self.model,self.test_loader)
+            print(f'Doing local model validation for collaborator {self.input}: {self.local_validation_score}')
+            self.next(self.join, exclude=['training_completed'])
+
+        @aggregator
+        def join(self,inputs):
+            self.average_loss = sum(input.loss for input in inputs)/len(inputs)
+            self.aggregated_model_accuracy = sum(input.agg_validation_score for input in inputs)/len(inputs)
+            self.local_model_accuracy = sum(input.local_validation_score for input in inputs)/len(inputs)
+            print(f'Average aggregated model validation values = {self.aggregated_model_accuracy}')
+            print(f'Average training loss = {self.average_loss}')
+            print(f'Average local model validation values = {self.local_model_accuracy}')
+            self.model = FedAvg([input.model for input in inputs])
+            self.optimizer = [input.optimizer for input in inputs][0]
+            self.current_round += 1
+            if self.current_round < self.rounds:
+                self.next(self.aggregated_model_validation, foreach='collaborators', exclude=['private'])
+            else:
+                self.next(self.end)
+            
+        @aggregator
+        def end(self):
+            print(f'This is the end of the flow')  
+
+
+You'll notice in the `FederatedFlow` definition above that there were certain attributes that the flow was not initialized with, namely the `train_loader` and `test_loader` for each of the collaborators. These are **private_attributes** that are exposed only throught he runtime. Each participant has it's own set of private attributes: a dictionary where the key is the attribute name, and the value is the object that will be made accessible through that participant's task.
+
+Below, we segment shards of the MNIST dataset for **four collaborators**: Portland, Seattle, Chandler, and Portland. Each has their own slice of the dataset that's accessible via the `train_loader` or `test_loader` attribute. Note that the private attributes are flexible, and you can choose to pass in a completely different type of object to any of the collaborators or aggregator (with an arbitrary name). These private attributes will always be filtered out of the current state when transfering from collaborator to aggregator, or vice versa.
+
+
+.. code-block:: python

```

Comment:
Here, the code probably needs to be updated. Check the updated example on:
https://github.com/securefederatedai/openfl/blob/develop/openfl-tutorials/experimental/Workflow_Interface_101_MNIST.ipynb
Also, be sure that other parts of the code examples are up to date.

Reply:
updated github.io code to reflect github tutorial updates